apiml-common/src/main/java/org/zowe/apiml/product/web/HttpConfig.java

```diff
@@ -177,8 +177,10 @@ public void init() {
             publicKeyCertificatesBase64 = SecurityUtils.loadCertificateChainBase64(httpsConfig);
 
         } catch (HttpsConfigError e) {
+            log.error("Invalid configuration of HTTPs: {}", e.getMessage());
             System.exit(1); // NOSONAR
         } catch (Exception e) {
+            log.error("Cannot construct configuration of HTTPs: {}", e.getMessage());
             apimlLog.log("org.zowe.apiml.common.unknownHttpsConfigError", e.getMessage());
```

Comment:
I think apimlLog is enough, actually the one on line 180 should be apimlLog as well, right?

Reply:
Honestly not. The main issue is that those cases end with `System.exit`. It is really bad solution and in case of a misconfiguration, it can lead to a frozen unit test. It is very difficult to find this location (I wasted that really huge amount of time). As you can see just one catch clause calls `apimlLog` and the log is even not logged during the testing.

The message is not helpful for customers, but definitely for developers. We should focus on removing `System.exit` in the whole project and then we could remove this message (again it is more complex).

Reply:
I think we should at least open issues in the repository keep track of the pending work.

Reply:
#2941

Reply:
then what is the reason to keep apimlLog?

Reply:
ok, removed